build.cake

```diff
@@ -130,6 +132,68 @@ void DoArchive(string runtime, DirectoryPath inputFolder, FilePath outputFile)
 }
 
 /// <summary>
+///  Generate the scripts which target the OmniSharp binaries.
+/// </summary>
+/// <param name="outputRoot">The root folder where the publised (or installed) binaries are located</param>
+void CreateRunScript(string outputRoot, string scriptFolder)
```

Comment:
`#load` and such is supported by cake, I'm fine with trying to split things up, as long as it makes sense to.

Making `build.cake` less daunting is not a bad thing.